backend/oasst_backend/api/v1/frontend_users.py

```diff
@@ -78,6 +78,7 @@ def query_frontend_user_messages(
     Query frontend user messages.
     """
     pr = PromptRepository(db, api_client)
+    pr.init_user(auth_method=auth_method, username=username)
```

Comment:
is `init_user` always coupled with creating a prompt repository? if yes, can we make it a part of PR's init function?

Reply:
it normally is but the call through init already is another way to specify the user, e.g. passing the protocol_user structure (including display_name but without the option to specify the user_id (UUID). The init_user() method is only used for the message query web-endpoints.